README.md

```diff
@@ -1,6 +1,227 @@
+SDK's are divided into two types:
+
+- **Management SDKs** - support creating services
+- **Instance SDKs** -  support interaction with services (limited to capabilities not supported by upstream tools)
+
+## Purpose of this repository
 
 Repository contains source code for openapi based generator along with automation scripts that generate all
-underlying RHOAS SDKS (<https://github.com/topics/rhoas-sdk>)
+underlying RHOAS SDKS. 
+
+# RHOAS SDK for Typescript and JavaScript
+
+Typescript and JavaScript packages and API clients for RHOAS services
+
+## Prequisites
+
+- [NodeJS 14.x lts](https://nodejs.org/en/about/releases/) or above
+
+Typically developers need to integrate with Management SDK's to create or manage service instances 
+and later use upstream libraries or management SDK's to interact with services.
+
+You can view all SDK packages directly in [npm](https://www.npmjs.com/search?q=keywords:rhoas)
+
+## Management SDK's
+
+| Name             | Package name                        | Status | Version            | NPM                 | Docs                |
+| ---------------- | :---------------------------------- | :----- | ------------------ | ------------------- | ------------------- |
+| kafka            | **@rhoas/kafka-management-sdk**     | beta   | ![kafkashield]     | [npm][kafkanpm]     | [doc][kafkagit]     |
+| service-accounts            | **@rhoas/service-accounts-sdk**     | alpha   | ![sashield]     | [npm][sanpm]     | [doc][sagit]     
+| connector        | **@rhoas/connector-management-sdk** | alpha  | ![connectorshield] | [npm][connectornpm] | [doc][connectorgit] |
+| service registry | **@rhoas/registry-management-sdk**  | alpha  | ![registryshield]  | [npm][registrynpm]  | [doc][registrygit]  |
+| smart events | **@rhoas/smart-events-management-sdk**  | alpha  | ![smarteventsshield]| [npm][smarteventsnpm]  | [doc][smarteventsgit] |
+
+
+## Instances SDK's
+
+| Name             | Package name                     | Status | Version             | NPM                 | Docs                |
+| ---------------- | :------------------------------- | :----- | ------------------- | ------------------- | ------------------- |
+| kafka            | **@rhoas/kafka-instance-sdk**    | beta   | ![kinstanceshield]  | [npm][kinstancenpm] | [doc][kinstancegit] |
+| service registry | **@rhoas/registry-instance-sdk** | beta   | ![srinstanceshield] | [npm][srinstancenpm]| [doc][srinstancegit]|
+
+## Docs
+
+[Documentation](/app-services-sdk-js/docs) 
+
+## Examples
+
+[Examples](/app-services-sdk-js/examples) 
+
+# RHOAS SDK for Python
+
+Python packages and API clients for Red Had OpenShift Application Services (RHOAS) 
+
+## Prequisites
+
+- [Python 3.9](https://docs.python.org/3/) or above
+- [pip](https://pypi.org/project/pip/) for installing packages
+
+## Installation
+
+Currently all RHOAS SDKs are bundled together. To install the RHOAS SDK with the pip package installer:
```

Comment:
this is actually surprising, should we have a ticket to improve here? Is Python the only exception?

Reply:
Yes currently python is the only one that combines them all, I don't think there was any plans to change this @dimakis when you created the python sdk was there a specific reason for this difference?

Reply:
discussed and agreed to consolidate those decisions while working on Kiota